core/common/src/geometry/AdditionalTransform.ts

```diff
@@ -84,6 +84,25 @@ export class Helmert2DWithZOffset implements Helmert2DWithZOffsetProps {
       Math.abs(this.rotDeg - other.rotDeg) < Geometry.smallAngleDegrees &&
       Math.abs(this.scale - other.scale) < Geometry.smallFraction);
   }
+
+  public convertHelmertToTransform(): Transform {
+    if (!this)
```

Comment:
Can `this` ever be falsey?

Reply:
Removed from this function. In users (like in transformer) will just need to account for the return value possibly being undefined